lib/manticore/client.rb

```diff
@@ -632,8 +632,8 @@ def setup_trust_store(ssl_options, context, trust_strategy)
         trust_store ||= blank_keystore
         open(ssl_options[:ca_file]) do |fp|
           cert_collection = CertificateFactory.get_instance("X509").generate_certificates(fp.to_inputstream).to_a
-          cert_collection.each do |cert|
-            trust_store.set_certificate_entry(cert.getSubjectX500Principal.name, cert)
+          cert_collection.each_with_index do |cert, i|
+            trust_store.set_certificate_entry("#{i}#" + cert.getSubjectX500Principal.name, cert)
```

Comment:
(I think this is a nice trick.)

Did you mean the entry's name to be `0#olleprincipal` and `1#olleprincipal`, etc?

Does the `#` between the index and the name carry any special meaning? Or could it be _any_ character?

To use the # between the index and the name:

```suggestion
            trust_store.set_certificate_entry("#{i}##{cert.getSubjectX500Principal.name}", cert)
```


Or, to use nothing in between: `0olleprincipal`, `1olleprincipal`

```suggestion
            trust_store.set_certificate_entry("#{i}#{cert.getSubjectX500Principal.name}", cert)
```

Reply:
> Did you mean the entry's name to be 0#olleprincipal and 1#olleprincipal, etc?

Indeed. I just chose the `+` syntax as the interpolated string would be a bit uglier to read, with the three #s.

> Does the # between the index and the name carry any special meaning? Or could it be any character?

The `#` has no practical value other than make is easier to read (a debatable call) the key entry. I'm not sure if all characters are allowed, but this one worked in my tests.